components/automate-chef-io/content/docs/nodes.md

```diff
@@ -57,9 +57,72 @@ The `/nodes` endpoint supports filtering by:
 - region
 - source_id (a reference to the primary provider's node)
 - state
-- statechange_timestamp
+- statechange_timerange (supports two timestamps of type "2019-03-05T00:00:00Z")
```

Comment:
Does the timestamp conform to any standard, eg ISO8601?

Reply:
urr..good question.

Reply:
RFC3339 is the date time standard that Chef tries to use.